environment.yml

```diff
@@ -4,7 +4,10 @@ channels:
 dependencies:
   # Required
   - python>=3.10
+  - IPython
```

Comment:
Note, don't include this in the conda recipe. It should be an optional dependency.